### ISSUE TITLE
TypeError: Cannot read property 'interceptorOptions' of undefined

### ISSUE THREAD

**Issue by barocsi** [voliva/angular2-interceptors#13]
Happens as
```
TypeError: Cannot read property 'interceptorOptions' of undefined
    at InterceptorService.request (x\node_modules\ng2-interceptors\lib\interceptor-service.js:94:44)
    at AuthHttp.requestWithToken (\x\node_modules\angular2-jwt\angular2-jwt.js:109:26)
    at MergeMapSubscriber.eval [as project] (x\node_modules\angular2-jwt\angular2-jwt.js:133:107)
```
angular2-jwt uses Http that is currently provided by ng2-interceptor. However seemingly it passes an undefined options argument that interceptor-service does not handle well.

**Comment by barocsi** [comment]
Bump

**Comment by voliva** [comment]
@barocsi Can you try if this is fixed on version 1.2.2 please?
(Commit is available here: https://github.com/voliva/angular2-interceptors/commit/9ac7470396c9ad00472048d5980b174048df34f9)

**Comment by barocsi** [comment]
Works thanks.
